mitodl/ocw-hugo-themes
# old course theme removed

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes https://github.com/mitodl/ocw-hugo-themes/issues/986

#### What's this PR do?
- Removes old course theme

#### How should this be manually tested?
- Checkout to [this](https://github.com/mitodl/ocw-hugo-projects/pull/236) branch in ocw-hugo-projects
- Point your `COURSE_HUGO_CONFIG_PATH` env var in `ocw-hugo-themes` to `course/config.yaml` instead of `course-v2/config.yaml`
- Build any course locally
- Verify that the course builds successfully
- Now test the course thoroughly and verify that everything is on point
- Repeat this for multiple courses
